code/modules/food_and_drinks/recipes/soup_mixtures.dm

```diff
@@ -1757,3 +1757,402 @@
+	name = "fox's prize soup"
+	icon = 'icons/obj/food/martian.dmi'
+	icon_state = "foxs_prize_soup"
+	drink_type = MEAT | VEGETABLES
+
+/datum/chemical_reaction/food/soup/foxs_prize_soup
+	required_reagents = list(
+		/datum/reagent/consumable/nutriment/soup/dashi = 30,
+		/datum/reagent/consumable/eggwhite = 10,
+	)
+	required_ingredients = list(
+		/obj/item/food/meat/cutlet/chicken = 1,
+		/obj/item/food/tofu = 1,
+	)
+	results = list(
+		/datum/reagent/consumable/nutriment/soup/foxs_prize_soup = 30,
+		/datum/reagent/consumable/nutriment/protein = 6,
+	)
+	percentage_of_nutriment_converted = 0.2
+
+// Secret Noodle Soup
+/datum/reagent/consumable/nutriment/soup/secret_noodle_soup
+	name = "Secret Noodle Soup"
+	description = "Made to a secret family recipe (that's in several cookbooks). What is the secret ingredient, you ask? Well, let's just say it could be anything..."
+	data = list("noodles" = 1, "chicken" = 1, "aromatic broth" = 1)
+	color = "#F0DD5A"
+
+/datum/glass_style/has_foodtype/soup/secret_noodle_soup
+	required_drink_type = /datum/reagent/consumable/nutriment/soup/secret_noodle_soup
+	name = "secret noodle soup"
+	icon = 'icons/obj/food/martian.dmi'
+	icon_state = "secret_noodle_soup"
+	drink_type = MEAT | VEGETABLES
+
+/datum/chemical_reaction/food/soup/secret_noodle_soup
+	required_reagents = list(
+		/datum/reagent/consumable/nutriment/soup/dashi = 30,
+	)
+	required_ingredients = list(
+		/obj/item/food/meat/cutlet/chicken = 1,
+		/obj/item/food/spaghetti/boilednoodles = 1,
+	)
+	results = list(
+		/datum/reagent/consumable/nutriment/soup/secret_noodle_soup = 30,
+		/datum/reagent/consumable/nutriment/protein = 6,
+	)
+	percentage_of_nutriment_converted = 0.2
+
+// Budae-Jjigae
+/datum/reagent/consumable/nutriment/soup/budae_jjigae
+	name = "Budae-Jjigae"
+	description = "A dish born of the American presence in South Korea, made with ingredients typical to the 1960s American army base- hot dogs, chap, and baked beans, as well as a number of native Korean ingredients such as gochujang and kimchi."
+	data = list("hot dog" = 1, "pork" = 1, "beans" = 1, "kimchi" = 1, "noodles" = 1)
+	color = "#F0DD5A"
+
+/datum/glass_style/has_foodtype/soup/budae_jjigae
+	required_drink_type = /datum/reagent/consumable/nutriment/soup/budae_jjigae
+	name = "budae-jjigae"
+	icon = 'icons/obj/food/martian.dmi'
+	icon_state = "budae_jjigae"
+	drink_type = MEAT | VEGETABLES | GRAIN
+
+/datum/chemical_reaction/food/soup/budae_jjigae
+	required_reagents = list(
+		/datum/reagent/water = 30,
+	)
+	required_ingredients = list(
+		/obj/item/food/canned/beans = 1,
+		/obj/item/food/spaghetti/rawnoodles = 1,
+		/obj/item/food/sausage/american = 1,
+		/obj/item/food/chapslice = 2,
+		/obj/item/food/kimchi = 1,
+		/obj/item/food/cheese/wedge = 1,
+	)
+	results = list(
+		/datum/reagent/consumable/nutriment/soup/budae_jjigae = 30,
+		/datum/reagent/consumable/nutriment/protein = 6,
+	)
+	percentage_of_nutriment_converted = 0.2
```

Comment:
For this one I might even suggest lowering it to `0.1` since it has so many ingredients. 
Not sure if you've tested this recipe to see how the numbers shape up.